src/ReverseProxy/Service/Config/ProxyRouteTransformExtensions.cs

```diff
@@ -171,6 +171,15 @@ public static void AddTransformForwarded(this ProxyRoute proxyRoute, bool useFor
             proxyRoute.Transforms.Add(transform);
         }
 
+        public static void AddTransformHttpMethod(this ProxyRoute proxyRoute, string fromHttpMethod, string toHttpMethod)
```

Comment:
Can you fill in some simple doc comments for these?

```suggestion
        /// <summary>
        /// Adds a transform to the route that will replace the HTTP method if it matches.
        /// </summary>
        public static void AddTransformHttpMethod(this ProxyRoute proxyRoute, string fromHttpMethod, string toHttpMethod)
```